jsf-showcase-webapp/src/test/java/com/liferay/faces/test/showcase/miscellaneous/MiscellaneousTester.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) 2000-2016 Liferay, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.liferay.faces.test.showcase.miscellaneous;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+
+import org.junit.Assert;
+
+import com.liferay.faces.test.selenium.Browser;
+import com.liferay.faces.test.selenium.assertion.SeleniumAssert;
+import com.liferay.faces.test.showcase.TesterBase;
+
+
+/**
+ * @author  Kyle Stiemann
+ * @author  Philip White
+ */
+public class MiscellaneousTester extends TesterBase {
+
+	protected void assertPortletContentEqual(Browser browser, String portletFileContent) {
```

Comment:
It looks like `assertPortletContentEqual()` and `assertWebappContentEqual()` could be combined into one method. Try to create a method like this: `assertElementContentEqualsFileContent(Browser browser, String elementXpath, String filePath)`.